src/azure-cli/azure/cli/command_modules/synapse/_params.py

```diff
@@ -115,10 +115,6 @@ def load_arguments(self, _):
                    help='The flag of enabling auto pause.')
         c.argument('delay', arg_group='AutoPause', help='The delay time whose unit is minute.')
 
-        # Environment Configuration
```

Comment:
Usually before we remove an argument, we add deprecate information as a warning for one or two sprints to notify users that the argument will be deprecated soon instead of a sudden removal which may break users' scripts.

Reply:
Hello @jsntcy , We've already tried it, this argument cannot work at all. https://github.com/Azure/azure-cli/issues/19342

Reply:
@zesluo, is this a behavior change from service side?

Reply:
It's a good practice to notify users for the breaking changes beforehand; otherwise when we release this change to users, the scripts with this logic will be broken.